### PR TITLE
i18n(fr): Add `deploy/cleavr.mdx` in French language

### PR DESCRIPTION
**Pull request by thomasbnt**
#### Description (required)

i18n(fr): Add `deploy/cleavr.mdx` in French language

I am participating in the Hacktoberfest! 🎃

- [i18n(fr): Add deploy/cleavr.mdx in French language](https://github.com/withastro/docs/commit/221472ad3ed1a36bb4cec8ba7b8d4b75f8c697b7)
#### Related issues & labels (optional)

Related to this issue https://github.com/withastro/docs/issues/4984, don't close it.

### CODE REVIEW

File: src/content/docs/fr/guides/deploy/cleavr.mdx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+## Prérequis
+
+Pour commencer, vous aurez besoin de
+
+- Un compte Cleavr
+- Un serveur sur votre fournisseur VPS utilisant Cleavr
+
+## Ajouter un site
+
+1. Dans Cleavr, naviguez au serveur auquel vous voulez ajouter votre projet Astro.
```
Comment (dreyfus92) [inline src/content/docs/fr/guides/deploy/cleavr.mdx:23]:
```suggestion
1. Dans Cleavr, naviguez vers le serveur auquel vous voulez ajouter votre projet Astro.
```

File: src/content/docs/fr/guides/deploy/cleavr.mdx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+2. Sélectionnez **Add Site** et remplissez les détails de votre application, comme le nom de domaine.
+
+3. Pour **App Type**, sélectionnez 'NodeJS Static' ou 'NodeJS SSR' en fonction de la façon dont vous configurez votre application Astro.
+
+4. Pour les applications statiques, définissez **Artifact Folder** à `dist`.
```
Comment (dreyfus92) [inline src/content/docs/fr/guides/deploy/cleavr.mdx:29]:
```suggestion
4. Pour les applications statiques, définissez **Artifact Folder** sur `dist`.
```

File: src/content/docs/fr/guides/deploy/cleavr.mdx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+5. Pour les applications SSR :
+- Définissez **Entry Point** à `entry.mjs`.
```
Comment (dreyfus92) [inline src/content/docs/fr/guides/deploy/cleavr.mdx:32]:
```suggestion
- Définissez **Entry Point** sur `entry.mjs`.
```

File: src/content/docs/fr/guides/deploy/cleavr.mdx [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+- Définissez **Artifact Folder** à `dist/server`.
```
Comment (dreyfus92) [inline src/content/docs/fr/guides/deploy/cleavr.mdx:33]:
```suggestion
- Définissez **Artifact Folder** sur `dist/server`.
```

**Review by dreyfus92 — CHANGES_REQUESTED** [review]
Bon boulot @thomasbnt, voici quelques mineur suggestions. Fais-moi savoir ce que tu penses.

**Comment by thomasbnt** [comment]
Merci @dreyfus92 pour les corrections !

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Great work 💜

**Pull request merged**